awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -1,35 +1,156 @@
+        {jobTemplate && (
+          <Detail
+            label={i18n._(t`Template`)}
+            value={
+              <Link to={`/templates/job_template/${jobTemplate.id}`}>
+                {jobTemplate.name}
+              </Link>
+            }
+          />
+        )}
+        <Detail label={i18n._(t`Job Type`)} value={toTitleCase(job.job_type)} />
+        {inventory && (
+          <Detail
+            label={i18n._(t`Inventory`)}
+            value={
+              <Link to={`/inventory/${inventory.id}`}>{inventory.name}</Link>
+            }
+          />
+        )}
+        {/* TODO: show project status icon */}
+        {project && (
+          <Detail
+            label={i18n._(t`Project`)}
+            value={<Link to={`/projects/${project.id}`}>{project.name}</Link>}
+          />
+        )}
+        <Detail label={i18n._(t`Revision`)} value={job.scm_revision} />
+        <Detail label={i18n._(t`Playbook`)} value={null} />
+        <Detail label={i18n._(t`Limit`)} value={job.limit} />
+        <Detail label={i18n._(t`Verbosity`)} value={VERBOSITY[job.verbosity]} />
+        <Detail label={i18n._(t`Environment`)} value={null} />
+        <Detail label={i18n._(t`Execution Node`)} value={job.exucution_node} />
+        {instanceGroup && (
+          <Detail
+            label={i18n._(t`Instance Group`)}
+            value={
+              <Link to={`/instance_groups/${instanceGroup.id}`}>
+                {instanceGroup.name}
+              </Link>
+            }
+          />
+        )}
+        {typeof job.job_slice_number === 'number' &&
+          typeof job.job_slice_count === 'number' && (
+            <Detail
+              label={i18n._(t`Job Slice`)}
+              value={`${job.job_slice_number}/${job.job_slice_count}`}
+            />
+          )}
+        {credentials && credentials.length > 0 && (
+          <Detail
+            fullWidth
+            label={i18n._(t`Credentials`)}
+            value={
+              <ChipGroup showOverflowAfter={5}>
+                {credentials.map(c => (
+                  <CredentialChip key={c.id} credential={c} isReadOnly />
+                ))}
+              </ChipGroup>
+            }
+          />
+        )}
+        {labels && labels.count > 0 && (
+          <Detail
+            fullWidth
+            label={i18n._(t`Labels`)}
+            value={
+              <ChipGroup showOverflowAfter={5}>
+                {labels.results.map(l => (
+                  <Chip key={l.id} isReadOnly>
+                    {l.name}
+                  </Chip>
+                ))}
+              </ChipGroup>
+            }
+          />
+        )}
+      </DetailList>
+      {job.extra_vars && (
+        <VariablesInput
+          css="margin: 20px 0"
+          id="job-variables"
+          readOnly
+          value={job.extra_vars}
+          rows={4}
+          label={i18n._(t`Variables`)}
+        />
+      )}
+      <VariablesInput
```

Comment:
Do we always show the Artifacts text editor? I'm wondering if we should conditionally render it similar to how you are currently doing it for extra_vars.